script-library/sshd-debian.sh

```diff
@@ -46,23 +45,27 @@ if ! dpkg -s openssh-server openssh-client > /dev/null 2>&1; then
     apt-get -y install --no-install-recommends openssh-server openssh-client 
 fi
 
+# Generate password if new password set to the word "random"
 if [ "${NEW_PASSWORD}" = "random" ]; then
     NEW_PASSWORD="$(openssl rand -hex 16)"
     EMIT_PASSWORD="true"
 fi
 
+# If new password not set to skip, set it for the specified user
 if [ "${NEW_PASSWORD}" != "skip" ]; then
     echo "${USERNAME}:${NEW_PASSWORD}" | chpasswd
     if [ "${NEW_PASSWORD}" != "root" ]; then
```

Comment:
@Chuxel, Should this be `${USERNAME}` instead of `${NEW_PASSWORD}`?

Reply:
Nope, if the new password is not set to skip, it pipes the username : password into chpasswd. See here for where the password would get set: https://github.com/microsoft/vscode-dev-containers/blob/master/script-library/docs/sshd.md#ad-hoc-usage

Reply:
Sorry, I mean on line 57, we compare `"${NEW_PASSWORD}" != "root"`, should that be `"${USERNAME}" != "root"`?